CONTRIBUTING.md

```diff
@@ -80,46 +121,16 @@ Where:
 - `[IssueSummary]` is a short summary of the issue topic, not including spaces, using Camel case or lower-case,
   dash-separated words. This summary, with its dash (‘-’) symbol prefix, is optional.
 
+## Important Notes
 
-## Contribution workflow
-
-Find an issue without the label `current sprint` and add a comment on it to inform the community that you are
-working on it.
-
-1. Make your [own fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) of the repository
-   target by the issue. Clone it on our local machine, then go inside the directory.
-
-2. We are working with [Pipenv](https://github.com/pypa/pipenv) for our virtualenv.
-   Create a local env and install development package by running `pipenv install --dev`, then run tests with `pipenv
-   run pytest` to verify your setup.
-
-3. For convention help, we provide a [pre-commit](https://pre-commit.com/hooks.html) file.
-   This tool will run before each commit and will automatically reformat code or raise warnings and errors based on the
-   code format or Python typing.
-   You can install and setup it up by doing:
-   ```bash
-   pipenv install pre-commit
-   pipenv run python -m pre-commit install
-   ```
-
-4. Make the changes.<br/>
-   You may want to also add your GitHub login as a new line of the `contributors.txt` file located at the root
-   of this repository. We are using it to list our contributors in the Taipy documentation
-   (see the "Contributing > Contributors" section) and thank them.
-
-5. Create a [pull request from your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).<br/>
-   Keep your pull request as __draft__ until your work is finished.
-   Do not hesitate to add a comment for help or questions.
-   Before you submit a pull request for review from your forked repo, check that it meets these guidelines:
-    - Include tests.
-    - Code is [rebase](http://stackoverflow.com/a/7244456/1110993).
-    - License is present.
-    - pre-commit works - without mypy error.
-    - GitHub's actions are passing.
-
-6. The taipy team will have a look at your Pull Request and will give feedback. If every requirement is valid, your
-   work will be added in the next release, congratulation!
+- If your PR is not created or there is no other activity within 14 days of being assigned to the issue, a warning message will appear on the issue, and the issue will be marked as "stale".
+- If your issue is marked as "stale", you will be unassigned after 14 days of inactivity.
```

Comment:
Stale seems a bit strong to me. What about "frozen"?

Reply:
What do you mean "frozen"? The label?

Reply:
Yes I am talking about the label. "Waiting for contributor" is good.
![image](https://github.com/Avaiga/taipy/assets/88036007/c787b99b-b2c7-4407-8c2a-c5521a1ec6cf)